include/exadg/structure/material/library/st_venant_kirchhoff.cpp

```diff
@@ -180,41 +204,75 @@ StVenantKirchhoff<dim, Number>::second_piola_kirchhoff_stress_symmetrize(
 
 template<int dim, typename Number>
 dealii::Tensor<2, dim, dealii::VectorizedArray<Number>>
-StVenantKirchhoff<dim, Number>::second_piola_kirchhoff_stress(
-  dealii::Tensor<2, dim, dealii::VectorizedArray<Number>> const & gradient_displacement,
-  unsigned int const                                              cell,
-  unsigned int const                                              q) const
+StVenantKirchhoffLargeDeformation<dim, Number>::second_piola_kirchhoff_stress(
+  tensor const &     gradient_displacement,
+  unsigned int const cell,
+  unsigned int const q) const
 {
-  if(large_deformation)
-  {
-    return (this->second_piola_kirchhoff_stress_symmetrize(
-      get_E<dim, Number>(get_F<dim, Number>(gradient_displacement)), cell, q));
-  }
-  else
-  {
-    return (this->second_piola_kirchhoff_stress_symmetrize(gradient_displacement, cell, q));
-  }
+  return (this->second_piola_kirchhoff_stress_symmetrize(
+    get_E<dim, Number>(get_F<dim, Number>(gradient_displacement)), cell, q));
```

Comment:
The `_symmetrize()` function has its origin in the linear case. Hence, it is not intuitive that the nonlinear case, where `E` is already symmetric, now also calls the `_symmetrize()` function.